### PR TITLE
Book

### PR DESCRIPTION
**Pull request by Danymations**
I wrote a little thing about Monika talking about originality.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4344,6 +4344,43 @@ label monika_confidence_2:
     m "Your everlasting love and care is just about all the support I need in order get to where I want to be."
     m "What kind of girlfriend would I be if I didn't return the favor~?"
     return
+    
+init 5 python:
+for key in ['book','tried','try','literature','original','originality']:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Indent this line. The `for` loop here is in the `init` block, so it needs to be indented once.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4344,6 +4344,43 @@ label monika_confidence_2:
...
+    m 1q "It was about a spider helping a pig thats going to die."
+    m 1n "Sound familiar...?"
+    m 1l "Ehe..."
+    m 1q "I felt pretty down when I found out."
+    m 1h "But it's okay."
+    m 2e "Sometimes you'll find out that your ideas aren't totally original."
+    m 1r "But...is anything ever truly original?"
+    m 2i "Have you heard of that show...Moon and the Forces of Good?"
+    m 2l "Is that what it's called?"
+         menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove one indent from the `menu` block (and code within the menu block). Here is a quick guide on indentation:

```
label this_is_label:
    m "i am dialogue"
    menu:
        m "I have question"
        "choice":
            m " I see you made choice"
        "other choice":
            m "I see you made other choice"
    m "more dialogue after choice"
    return
```

**Comment by Danymations** [comment]
Okay, done.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4344,6 +4344,43 @@ label monika_confidence_2:
...
+    m 1n "Sound familiar...?"
+    m 1l "Ehe..."
+    m 1q "I felt pretty down when I found out."
+    m 1h "But it's okay."
+    m 2e "Sometimes you'll find out that your ideas aren't totally original."
+    m 1r "But...is anything ever truly original?"
+    m 2i "Have you heard of that show...Moon and the Forces of Good?"
+    menu:
+        m 2l "Is that what it's called?"
+        "Yes."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu` block choices need a colon after them like this:
```
menu:
    m "make a choice"
    "Choice":
        m "good choice"
    "other choice":
        m "also good choice"
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4344,6 +4344,43 @@ label monika_confidence_2:
...
+             m 2e "Ehehe...are you sure?"
+             m 2b "Well, I'll trust your judgement!
+         "No."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Refer to above comment

**Comment by Danymations** [comment]
Super sorry! Now it should be done.

**Comment by Rune0n** [comment]
Closing for now. A re-write is encouraged. Some comments below.
-Considering the other books that have shown up in conversations, the chance of Monika not knowing about "Charlotte's Web" is exceedingly low.
-Unsure if this was intentional or not, but wasn't it "Star vs the Forces of Evil"?
-Pretty evident contradiction/lack of clarity in the topic.
>m 1r "But...is anything ever truly original?"
>m "Not everything turns out totally original."
>m "You can create something and turn it into something original."
Rewording is advised.

**Pull request closed without merging**